src/Domain/Tags/ElementDistribution.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <optional>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/ElementDistribution.hpp"
+#include "Options/Auto.hpp"
+#include "Options/String.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain {
+namespace OptionTags {
+/// \ingroup OptionTagsGroup
+/// \ingroup ComputationalDomainGroup
+struct ElementDistribution {
```

Comment:
Should this go into the `ResourceInfo` group? I can't decide whether or not I like that. I don't love having a bunch of lowest-level options if possible :Shrug:

Reply:
`ResourceInfo` isn't actually an options group, it's just a struct. That being said, I can put it into the struct and just add a member function for accessing it if you think that's better. The `ResourceInfo` is available in the initialization items, so I don't have strong opinions where we place the option

Reply:
Ah okay. Hmm, yea, if it's easy to do maybe it's worth doing? My only hesitation is then if we have an executable that uses ResourceInfo but no elements things will be confusing. In a sense we probably want something like a `Parallelization` group, but that seems beyond the scope of this PR and should probably have input from the other core devs :shrug: So I will let you make a decision on what you think is best and we'll just go with that :)

Reply:
Oh actually, this is true for the CCE executable. It doesn't have a `DgElementArray`, so with the current setup you never specify an element distribution. But if it was in the `ResourceInfo`, then you'd have to, but it'd be a useless option.

I like the idea of a `Parallelization` group. I'll put the element distribution in there, and we can add to it later with whatever else we come up with.

I guess we could technically put the `ResourceInfo` into `Parallelization`, but I think that's out of the scope of this PR.

Reply:
That sounds like a good plan to me!